darwin/client.py

```diff
@@ -353,30 +366,43 @@ def create_dataset(self, name: str, team: Optional[str] = None) -> RemoteDataset
 
     def fetch_remote_classes(self, team: Optional[str] = None) -> Optional[Dict[str, Any]]:
         """Fetches all remote classes on the remote dataset"""
-        team = self.config.get_team(team or self.default_team)
+        the_team: Optional[Team] = self.config.get_team(team or self.default_team)
```

Comment:
I would actually rename the argument to be `team_name` or `team_slug` (depending on what this string actually represents), so this one can be named `team`.

I know it's breaking it for people who call `fetch_remote_classes(team=team)`, but I think it's fine for a minor upgrade

Reply:
Same in other functions below.